lunavl/sdk/estimators/face_estimators/mask.py

```diff
@@ -139,31 +131,29 @@ class MaskEstimator(BaseEstimator):
     Warp mask estimator.
     """
 
-    #  pylint: disable=W0235
-    def __init__(self, maskEstimator: IMedicalMaskEstimatorPtr):
```

Comment:
for what this change?

Reply:
Overriding this method is superfluous. This logic is used in the base class

Reply:
Base class takes any estimator. this class takes only IMedicalMaskEstimatorPtr. this are reflected in the typing.

Reply:
I returned this method. Such as same method in credibilityCheck (https://github.com/matemax/lunasdk/pull/108#discussion_r585173082 https://github.com/matemax/lunasdk/pull/108#discussion_r585173821)